main/src/library/Http.flix

```diff
@@ -0,0 +1,48 @@
+/*
+ *  Copyright 2024 Holger Dal Mogensen
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+///
+/// The effect used to interact with the HTTP protocol.
+///
+eff Http {
+    pub def get(url: String, headers: Map[String, String]): Http.Response
```

Comment:
I think we also want: head, post, put, and delete

Reply:
Possibly that could wait till iteration 2

Reply:
Alternatively, we could add an `HttpMethod` enum and take it as a parameter, while renaming the funciton to `request`.

Reply:
Lets keep it like this for now.